RyanNoelk/OpenEats
# Feature/mobile browse

Giving the mobile site a face-lift.

To Test: 
- Run through the site in desktop and mobile and make sure the browse sections still works as expected.
